---
title: "Target details for \n for FDA drug approvals \n 2025"
date: "Last update: `r Sys.setlocale('LC_TIME', 'C') ; format(Sys.time(), '%d, %B, %Y')`"    
output:
  html_document:
    toc: true
    fig_caption: true
bibliography: fda.bib
biblio-style: apalike


---
<br>
<b>DISGENET  is provided by [Medbioinformatics Solutions S.L.](https://www.medbioinformatics.com/)</b>   
Rambla de CataluÃ±a 14, 7Âº 1Âª 
08007  Barcelona (Spain)   
<b>Contact us at </b> info@disgenet.com   



```{r set-options, echo=FALSE, cache=FALSE}
knitr::opts_chunk$set(
  eval=TRUE
)
htmltools::img(src = knitr::image_uri(file.path(".", "images/logo_disgenet.png")), 
               alt = 'logo', 
               style = 'position:absolute; top:0; right:0; padding:10px;')
```

# Introduction



```{r,  echo=FALSE, warning=F, message=FALSE, eval=TRUE, cache=F}
rm(list =ls())
library(data.table)
library(DT)
library(tidyr)
library(dplyr)
library(httr)
library(jsonlite)
library(kableExtra)
library(sysfonts)
library(showtext)
library(ggplot2)
library(ggrepel)
library(scales)
library(plotly)

font_add_google("Poppins", "poppins")
showtext_auto()
createLink <- function(chemblid){
  sprintf("<a href='https://www.ebi.ac.uk/chembl/compound_report_card/%s' target='_blank'>%s</a>",chemblid, chemblid)
}
 
fda <- fread("data/nrdd_table_2025.tsv", skip = 2)
fda <- janitor::clean_names(fda)

 

```

Target selection remains a key determinant of success in pharmaceutical R&D, as efficacy and safety issues continue to drive late-stage failures. In 2025, the FDA approved `r length(unique(fda$drug_brand_name))` drugs [@Mullard2026] (Table 1), including 38 protein-targeted therapies and three mRNA-based treatments, offering a real-world snapshot of which targets actually make it to the finish line. By integrating data on disease associations from [DISGENET](https://www.disgenet.com/) [@Pinero2026], gene expression, and historical target status, we explored what these approvals can teach us about choosing the right targets.



```{r, results='asis', warning= F, echo=FALSE, message=FALSE ,fig.width= 10, fig.height= 10, eval=T}

results_table <- fda   %>% select(drug_brand_name, sponsor,properties, indication ) %>% unique()
DT::datatable(results_table, caption = "Table 1: 2025 FDA approvals" ,  filter = 'top', options = list(pageLength = 10,  autoWidth = F), rownames = F)



results_table <- fda %>%
  select(drug_brand_name, sponsor, properties, indication) %>%
  distinct() %>%
  mutate(
    drug_brand_name = sub("a,b$", "", drug_brand_name)
  ) %>%
  mutate(
    drug_brand_name = sub("a$", "", drug_brand_name)
  )

DT::datatable(
  results_table,
  caption = "Table 1: 2025 FDA approvals",
  filter = "top",
  options = list(pageLength = 10, autoWidth = FALSE),
  rownames = FALSE
) %>%
  formatStyle(
    "drug_brand_name",
    target = "row",
    backgroundColor = styleEqual(
      c("Gepotidacin (Blujepa)", "Zoliflodacin (Nuzolvence)", "Clesrovimab (Enflonsia)"),
      c("#fff3cd", "#fff3cd", "#fff3cd")  # light yellow highlight
    )
  )

```

# From Legacy Targets to Emerging Opportunities


All drug targets are represented in DISGENET, underscoring their established relevance to human disease. Most are associated with hundreds of disease phenotypes. All are referenced in hundreds, often thousands, of publications in the disease genomics literature, reflecting decades of accumulated biological and clinical knowledge. Consistent with this depth of evidence, many 2025 FDA approvals focus on long-studied targets such as EGFR, ERBB2, and ESR1, which exhibit publication histories dating back to the 1970s and broad links to diverse disease areas.

```{r, results='asis', warning= F, echo=FALSE, message=FALSE ,fig.width= 6, fig.height= 6, eval=T}

target_summary <- fread("data/target_summary.tsv")
DT::datatable(target_summary, caption = "Table 2: Target Summary" ,  filter = 'top', options = list(pageLength = 10,  autoWidth = F), rownames = F)


p <- ggplot(
  target_summary,
  aes(
    x = min_year_pmid,
    y = n_diseases,
    size = n_pmids,
    text = paste0(
      "Gene: ", GENE_SYMBOL,
      "<br>First PMID year: ", min_year_pmid,
      "<br>Diseases: ", n_diseases,
      "<br>PMIDs: ", scales::comma(n_pmids)
    )
  )
) +
  geom_point(
    alpha = 0.8,
    color = "#d02591"
  ) +
  scale_size_continuous(
    name   = "Number of PMIDs",
    range  = c(2, 10),
    breaks = c(100, 500, 1000, 10000, 50000),
    labels = scales::comma
  ) +
  scale_x_continuous(
    limits = c(1960, 2005),
    breaks = seq(1960, 2025, by = 10)
  ) +
  labs(
    x = "Year of first PMID association",
    y = "Number of associated diseases"
  ) +
  theme_minimal(base_family = "poppins")

ggplotly(p, tooltip = "text")

```


# Translational and Disease Insights

DISGENET also shows that most of the approved targets appear in clinical trial contexts. Examples include patient stratification (e.g., JAK2, MET) and monitoring pharmacodynamic response (e.g., APOC3, PCSK9, BTK), reflecting their translational relevance throughout drug development.


```{r, results='asis', warning= F, echo=FALSE, message=FALSE ,fig.width=6, fig.height= 6, eval=T}

ct_summary <- fread("data/ct_summary.tsv")
ct_summary <- rename(ct_summary, N.Clinical.Trials  =nct , N.Diseases.CTs = ndis)
DT::datatable(
  ct_summary,
  caption = "Table 2: Clinical Trials info for targets of 2025 FDA approvals",
  filter = "top",
  options = list(
    pageLength = 10,
    autoWidth = TRUE
  ),
  rownames = FALSE,
  width = "600px"   # ðŸ‘ˆ make table narrower
)


yearly_counts <- fread("data/ct_yearly_counts.tsv")
genes_of_interest <- c("APOC3", "BTK", "JAK2", "MET", "PCSK9", "MYBPC3", "EDNRA", "PDE4B" )
yearly_counts <- yearly_counts[GENE_SYMBOL %in% genes_of_interest]

 
label_data <- yearly_counts[
  ,
  .SD[.N],
  by = GENE_SYMBOL
]


p <- ggplot(
  yearly_counts,
  aes(
    x = ASSOC_REFERENCE_YEAR,
    y = cumulative_trials,
    color = GENE_SYMBOL,
    group = GENE_SYMBOL,
    text = paste0(
      "Gene: ", GENE_SYMBOL,
      "<br>Year: ", ASSOC_REFERENCE_YEAR,
      "<br>Cumulative trials: ", comma(cumulative_trials)
    )
  )
) +
  geom_line(linewidth = 1) +
  geom_point(size = 2) +
  labs(
    x = "Year",
    y = "Cumulative number of clinical trials"
  ) +
  scale_x_continuous(
    expand = expansion(mult = c(0.02, 0.12))
  ) +
  theme_minimal(base_size = 18) +
  theme(
    legend.position = "none",
    axis.title = element_text(size = 14, face = "bold"),
    axis.text  = element_text(size = 14)
  )

ggplotly(p, tooltip = "text")
 


```

# Is target  promiscuity  really a problem?

To better understand why many targets are associated with a large number of diseases, we examined disease-class enrichment for each gene. Instead of counting diseases, we looked at whether a targetâ€™s associations repeatedly fall within the same therapeutic areas compared with what is typically seen across all genes in DISGENET. This shows that these broad associations are often concentrated within a small number of related disease classes that, in most cases, include the therapeutic indication of the target.

Representative examples illustrate this pattern (Figure 3). PCSK9 is linked to many diseases, but these are mostly related to hypercholesterolemia and related cardiovascular or metabolic disorders. Similarly, IL5 is enriched in immune, hematologic, and respiratory disease classes, consistent with its role as the target of depemokimab (Exdensur) for severe eosinophilic asthma. Together, these examples show that many targets are not broadly associated with unrelated diseases, but instead display focused association patterns within biologically and therapeutically coherent disease areas. This analysis challenges the notion that broad disease associations necessarily imply poor target specificity, highlighting instead that specificity often emerges at the level of therapeutic areas rather than individual diseases.


```{r, results='asis', warning= F, echo=FALSE, message=FALSE ,fig.width=6, fig.height= 4, eval=T}
enrich_dt <- fread("data/target_enrichment.tsv")

 ggplot(enrich_dt[ enrich_dt$Symbol %in% c("PCSK9", "IL5")], aes(
  x = disease_class_name,
  y = log2_enrich,
  fill = log2_enrich > 0
)) +
  geom_col(show.legend = FALSE) +
  facet_wrap(~ Symbol, ncol = 3) +
  scale_fill_manual(values = c("#2C7BB6", "#D7191C")) +
  geom_hline(yintercept = 0, linetype = "dashed") +
  theme_minimal() +
  theme(
    axis.text.x = element_text(size = 14, hjust = 1),
      strip.text = element_text(size = 14, face = "bold") ,  
        axis.text.y = element_text(size = 18)
  ) +
  labs(y = "log2 enrichment", x = "") + coord_flip()

```




```{r, results='asis', warning= F, echo=FALSE, message=FALSE ,fig.width=6, fig.height= 6, eval=T}
```
# References
